cypress/integration/circulation/settings/circulation.settings.createPatronNoticePolicy.spec.js

```diff
@@ -0,0 +1,22 @@
+import { TextField, Button, TextArea } from '../../../../interactors';
+import TopMenu from '../../../support/fragments/topMenu';
+import TestType from '../../../support/dictionary/testTypes';
+import SettingsCirculations from '../../../support/fragments/circulation/settingsCirculations';
+import NewPatronNoticePolicies from '../../../support/fragments/circulation/newPatronNoticePolicies';
+
+describe('ui-circulation-settings: create patron notice policies', () => {
+  const patronNoticePolicy = { ...NewPatronNoticePolicies.defaultUiPatronNoticePolicies };
+  beforeEach('login', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(`${TopMenu.settingsCirculation}${SettingsCirculations.settingsCirculationPath.patronNoticePolicies}`);
```

Comment:
move whole path into PatronNoticePoliciesSettings fragment and use it instead of such combination